src/main/webapp/js/omniComment.js

```diff
@@ -86,16 +86,16 @@ $(document).ready(function() {
             $('#no-comment-panel').show();
             // if all is checked, show giver and status for better user experience
             if (!$('#panel_all').prop('checked')) {
-                $('#giver_all').parent().parent().hide();
-                $('#status_all').parent().parent().hide();
+                $('#giver_all').closest('.col-sm-4').hide();
```

Comment:
Same here, for these and the rest below do we have a better class or an ID to select? It's not that safe to select based on a bootstrap class as future HTML changes can easily break this selection since column classes in CSS libraries are often used for layout